src/icons/GPT.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import {useTheme} from '@gravity-ui/uikit';
+
+const GPTIcon = () => {
```

Comment:
On the one hand, code duplication has been eliminated. On the other hand, it is not an icon anymore, but a component that accesses the context and is quite smart. I suggest using 2 SVG icons - dark and light - instead of this solution, and calling one or the other depending on the theme.

Reply:
Or let's just remove the connection with useTheme and drop the props so that the component is simple

https://github.com/gravity-ui/markdown-editor/blob/de549914279be23f6b45dda50aec5ba7fdb7c439/src/icons/WysiwygMode.tsx#L3

Then the place call can be something like this:

```
// outside component:
const GPTDarkIcon = () => (<GPTIcon theme="dark" />;
const GPTLightIcon = () => (<GPTIcon theme="light" />;
...
// inside component
const theme = useTheme()
...
icon={<Icon data={theme === 'dark' || theme === 'dark-hc' ? GPTDarkIcon : GPTLightIcon} />}
```